sbt-plugin/src/main/scala/com/akkaserverless/sbt/AkkaserverlessPlugin.scala

```diff
@@ -48,6 +48,8 @@ object AkkaserverlessPlugin extends AutoPlugin {
       "These are the source files that are placed in the source tree, and after initial generation should typically be maintained by the user.\n" +
       "Files that already exist they are not re-generated.")
     val temporaryUnmanagedDirectory = settingKey[File]("Directory to generate 'unmanaged' sources into")
+    val onlyUnit = settingKey[Boolean]("Filters out integration tests. By default: true")
```

Comment:
I think the default should be `false`, and that we also read the setting from a System.property so that it can be used like this:
```
sbt -DonlyUnitTest=true
```

Reply:
yep, sounds good.

Reply:
OK, I talked with @raboof and it seems the main problem was trying to add `it:test` through `Defaults.itSettings`. This wouldn't allow further configuring the project.